docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1144,6 +1184,9 @@ function setTranscriptEvents() {
     meeting.on('meeting:receiveTranscription:stopped', () => {
       generalToggleTranscription.innerText = "Start Transcription";
       generalTranscriptionContent.innerHTML = 'Transcription Content: Webex Assistant must be enabled, check the console!';
+      generalHighlightTranscription.setAttribute('data-enabled', "false");
```

Comment:
**IMPORTANT** - Here we need to disable the transcription button and replace the text as when transcription is stopped highlighting no longer sends events.
Also - this gets triggered when we close the meeting also.

I'm not a 100% sure about this method (open to any suggestions). However, since this works for the sample app - I feel it should work.